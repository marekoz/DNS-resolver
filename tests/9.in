-s 1.1.1.1 -r -x 1.1.1.1