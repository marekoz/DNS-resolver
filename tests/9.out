Question section (1)
  1.1.1.1.in-addr.arpa., Type: PTR, Class: IN
Answer section (1)
  1.1.1.1.in-addr.arpa., Type: PTR, Class: IN, TTL: <number>, Data length: 17, one.one.one.one.
Authority section (0)
Additional section (0)